src/sentry/snuba/metrics/query_builder.py

```diff
@@ -53,7 +54,7 @@
 
 
 def parse_field(field: str) -> MetricField:
-    matches = FIELD_REGEX.match(field)
+    matches = PUBLIC_NAME_REGEX.match(field)
```

Comment:
do we want to write new unittests? Or do you think we rely on old uint tests and it's enough?

Reply:
yeah I agree .. will add some unit tests